README.md

```diff
@@ -1,2 +1,248 @@
-# web-devops-roadmap
-DevOps Roadmap for web developers
+# Web DevOps Roadmap
+
+This repository contains initial DevOps roadmap for all the web developers. Anyone can start this training with their existing learning projects. The approximate time for complete these practicals is 10 days.
```

Comment:
This repository contains a basic DevOps roadmap for all web developers. Anyone can start this training with their existing learning projects. The approximate duration to complete this roadmap is 10 days, it can vary based on experience though.